safe/impact_functions/generic/classified_polygon_people/test/test_classified_polygon_people.py

```diff
@@ -55,6 +55,7 @@ def test_run(self):
         rect_extent = [
             extent.xMinimum(), extent.yMaximum(),
             extent.xMaximum(), extent.yMinimum()]
+        print rect_extent
```

Comment:
We should use 'prints' in production code (probably in non production code too since we have logging framework). Search InaSAFE code for 'LOGGER' for examples of how to use. Log messages will show in QGIS log panel.